tests/extmod/time_time_ns.py

```diff
@@ -22,3 +22,9 @@
     print(True)
 else:
     print(t0, t1, t1 - t0)
+
+# Check that time.time() and time.time_ns() are within a second of each other.
+for _ in range(10):
+    t_s, t_ns = time.time(), time.time_ns()
```

Comment:
are there any ports where `time.time()` result is an int not a float?

Reply:
Yes, the unix and webassembly ports.  Will fix.

Reply:
Actually, I think it's fine as-is, regardless of whether `time()` returns a float or int.  If it's a float then multipliying by 1_000_000_000 yields a float, then the subtraction is a float and so is the comparison.  And it passes.

But that can loose precision going to float (double).  I have tweaked the test a bit.  I don't want to rely on the port having float at all, but it should work in all cases.  See what you think.